super-csv/src/test/java/org/supercsv/io/TokenizerTest.java

```diff
@@ -586,6 +587,178 @@ public void testQuotedSpaces() throws Exception {
 		assertEquals("   three   ", columns.get(2));
 		assertEquals(input, tokenizer.getUntokenizedRow());
 	}
+
+	/**
+	 * Tests the readColumns() method with a quote character escaped with another quote (RFC style)
+	 */
+	@Test
+	public void testQuotedQuoteCharWithDoubleQuoteEscape() throws Exception {
+		final String input = "one,\"two\",\"field with \"\" quote char\",four";
+		tokenizer = createTokenizer(input, NORMAL_PREFERENCE);
+		tokenizer.readColumns(columns);
+
+		assertTrue(columns.size() == 4);
+		assertEquals("one", columns.get(0));
+		assertEquals("two", columns.get(1));
+		assertEquals("field with \" quote char", columns.get(2));
+		assertEquals("four", columns.get(3));
+		assertEquals(input, tokenizer.getUntokenizedRow());
+	}
+
+	/**
+	 * Tests the readColumns() method with a quote character escaped with another quote (RFC style)
+	 */
+	@Test
+	public void testQuotedQuoteCharWithBackslashEscape() throws Exception {
```

Comment:
(Javadoc was misleading, correcting..)

This test checks with a backslash escape, whereas the other test checks with a doubled quote escape.  The difference is for `quoteEscapeChar == quoteChar` vs `!=`.  The reason for both tests is that the doubled quote character caused some problems with my earlier implementation so I wanted to make sure to cover both cases in testing now.